packages/widget-chat/src/modules/redux-spark/spark.js

```diff
@@ -3,6 +3,8 @@
 import '@ciscospark/plugin-avatar';
 import '@ciscospark/plugin-conversation';
 import '@ciscospark/plugin-flag';
+import '@ciscospark/plugin-mercury';
+import '@ciscospark/plugin-wdm';
```

Comment:
Did we actually need to add these imports to get the tests working? If not, we may want to ditch them.